identifyyeastcells.py

```diff
@@ -0,0 +1,1146 @@
+            "quality" measure. Use this option if you want to absolutely prohibit too small (or too big) cells to be chosen (regardless of "quality"). 
+            """%globals())
+
+        self.min_cell_area = cps.Integer(
+            "Minimal area of accepted cell in pixels",
+            900, minval=10, doc ='''\
+            <i>(Used only when you want to filter cells based on area)</i><br>
+            The minimum cell area is used while final filtering of cells. 
+            Please use e.g. ImageJ to measure the average cell size in pixels.
+            '''%globals()
+            )
+
+        self.max_cell_area = cps.Integer(
+            "Maximum area of accepted cell in pixels",
+            5*900, minval=10, doc ='''\
+            <i>(Used only when you want to filter cells based on area)</i><br>
+            The maximum cell area is used while final filtering of cells. 
+            Please use e.g. ImageJ to measure the average cell size in pixels.
+            '''%globals()
+            )
+
+        
+        self.background_brighter_then_cell_inside = cps.Binary(
+            'Is the area without cells (background) brighter then cell interiors?', True, doc="""
+            Please check if the area inside of the cells is <b>darker</b> then the area without the cells (background). Use e.g. ImageJ to measure 
+            average intensity.
+            """%globals()
+            )
+
+        self.bright_field_image = cps.Binary(
+            'Do you want to segment brightfield images?', True, doc="""
+            Choose this option if you want to segment a brightfiled image. For segmentation of fluorescent images please answer "No". 
+            """%globals()
+            )
+
+        self.advanced_parameters = cps.Binary(
+            'Use advanced configuration parameters', False, doc="""
+            Do you want to use advanced parameters to configure plugin? They allow for more flexibility,
+            however you need to know what you are doing.
+            """%globals()
+            )
+
+        self.segmentation_precision = cps.Integer(
+            "Segmentation precision",
+            2,minval=1,maxval=5,doc = '''\
+            <i>(Used only when you want to specify advanced parameters)</i><br>
+            Describes how thouroughly the algorithm searches for cells. Higher values should 
+            make it easier to find smaller cells because the more parameters sets are searched. 
+            The cost is longer runtime.
+            '''%globals()
+            )
+
+        self.specify_precision_details = cps.Binary(
+            '''\
+            <i>(Used only when you want to specify advanced parameters)</i><br>
+            Do you want to edit details of segmentation precision?<br>
+            <img src="memory:%(TECH_NOTE_ICON)s">&nbsp;These are low level parameters that can be modified
+            to further tweak result.
+            ''', False)
```

Comment:
Missing `%globals()`.

Alternatively (and preferably) use [format](https://pyformat.info/).